code/datums/wounds/cranial_fissure.dm

```diff
@@ -33,6 +33,8 @@
 	severity = WOUND_SEVERITY_CRITICAL
 	sound_effect = 'sound/effects/dismember.ogg'
 
+	treatable_by = list(/obj/item/kitchen/spoon)
```

Comment:
Should organ bag also be valid for mediborgs too?

Reply:
Mediborgs don't eat from bowls of icecream and thus are exempt from being able to expertly scoop out brains.

Come to think of that hemostats should be made to work however, what's your opinion?

Reply:
Trying to stick to the idea that removing brains should be harder, hemostat is something many people have by default, spoons aren't. Mediborgs I thought would be good to include here because it's either an intentional setup with work put into it, or just brain transfer